### DISCUSSION TITLE
あなたの日本で一番好きな場所は？ Where is your favorite place in Japan

### DISCUSSION THREAD

**Discussion by seiji-shimizu** [NAIST-SE/AttractingContributors#549] [Q&A]
https://github.com/seiji-shimizu/AttractingContributors

**Comment by XC1996114514** [comment]
下北沢です

**Comment by ani-hovhannisyan** [comment]
I contributed, please can you do it back?
Discussion https://github.com/NAIST-SE/AttractingContributors/discussions/543#discussion-4155998
My project: https://github.com/ani-hovhannisyan/AttractingContributors-Motorcycle-Bicycle
Thanks in advance
